lua/lint/linters/credo.lua

```diff
@@ -0,0 +1,9 @@
+return {
+  cmd = 'mix credo',
+  args = { '--strict' },
```

Comment:
```suggestion
  cmd = 'mix',
  args = { 'credo', '--strict' },
```

`cmd` must be a single command